app/src/main/java/com/framgia/fsalon/utils/Utils.java

```diff
@@ -107,4 +114,28 @@ public static String getStatus(int status) {
                 return Constant.BookingStatus.NA;
         }
     }
+
+    public static void requestCallPermission(final Context context) {
+        if (ActivityCompat
+            .shouldShowRequestPermissionRationale((Activity) context, Manifest.permission
+                .CALL_PHONE)) {
+            AlertDialog.Builder builder = new AlertDialog.Builder(context);
+            builder
+                .setMessage(R.string.msg_call)
+                .setPositiveButton(R.string.action_ok, new DialogInterface.OnClickListener() {
+                    @Override
+                    public void onClick(DialogInterface dialog, int which) {
+                        ActivityCompat.requestPermissions((Activity) context,
+                            new String[]{Manifest.permission
+                                .CALL_PHONE}, Constant.REQUEST_CALL_PERMISSION);
+                    }
+                })
+                .setNegativeButton(R.string.action_close, null);
+            builder.create().show();
+        } else {
+            ActivityCompat.requestPermissions((Activity) context,
+                new String[]{Manifest.permission
+                    .CALL_PHONE}, Constant.REQUEST_CALL_PERMISSION);
```

Comment:
ERROR: 'array initialization' child have incorrect indentation level 20, expected level should be one of the following: 24, 28, 33, 36.